test/fixtures/valid_manifest.json

```diff
@@ -26,6 +26,18 @@
         "es": "Perdón, no estoy seguro de lo que necesitás."
       }
     },
+    "unsubscribe": {
+      "message": {
+        "en": "Send UNSUBSCRIBE to stop receiving messages",
+        "es": "Enviá DESUSCRIBIR para dejar de recibir mensajes"
+      }
+    },
+    "unsubscribe_keyword": {
+      "message": {
```

Comment:
This `message` property shouldn't be there. Look at the `keyword_responder` `keywords` property for instance.